utils/build/docker/nodejs/express4/iast/index.js

```diff
@@ -261,6 +265,34 @@ function initRoutes (app, tracer) {
     res.send('OK')
   })
 
+  function searchLdap(filter, req, res) {
+    const sendError = (err) => res.status(500).send(`Error: ${err}`) 
+
+    ldap.connect()
+      .catch(sendError)
+      .then(client => 
+        client.search('ou=people', filter, (err, searchRes) => {
+          if (err) return sendError(err)
+
+          const entries = new Set()
+          searchRes.on('searchEntry', entry => entries.add(entry.json));
```

Comment:
Remove semicolon `;`

Future work: We should add linter to this nodejs project ...